core/commonMain/src/ArgType.kt

```diff
@@ -75,18 +75,21 @@ abstract class ArgType<T : Any>(val hasParameter: kotlin.Boolean) {
          * Helper for arguments that have limited set of possible values represented as enumeration constants.
          */
         inline fun <reified T: Enum<T>> Choice(
-            noinline toVariant: (kotlin.String) -> T = { enumValueOf(it.toUpperCase()) },
+            noinline toVariant: (kotlin.String) -> T = {
+                enumValues<T>().associateBy { it.toString().toLowerCase() }[it.toLowerCase()] ?:
```

Comment:
`associateBy` seems to be a wasted work here, because the resulting map is discarded right after the first use. Better then just to use `find { e -> e.toString().equals(it, ignoreCase = true) }`.